.testing/Makefile

```diff
@@ -321,7 +328,7 @@ results/%/ocean.stats.restart: ../build/symmetric/MOM6
 clean: clean.stats
 	@# Assert that we are in .testing for recursive delete
 	@[ $$(basename $$(pwd)) = .testing ]
-	rm -rf ../build
+	rm -rf ../build ../deps/fms
```

Comment:
In nearly every case, it's not necessary to clear the FMS build, so I've been reluctant to include this.  I'd rather find a way to get the Makefile to be smart enough to detect if FMS source has changed.  And rebuilding FMS can be expensive.

A big bottleneck here is that this Makefile doesn't see the changes detected by the actual FMS makefile from mkmf.  I currently don't know how to solve that.

Reply:
That's fine. It's just for interactive we sometimes need to do a real clean. In the old days we used "clean" "Clean" and "CLEAN" for degrees of cleaning.